cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -176,12 +176,14 @@ This table describes the available query options for the connect-components plan
 
 | connectComponentsPlanner=greedy
 | Use a greedy approach when combining sub-plans.
-| X
+[NOTE]
+Using this option can significantly _reduce_ the planning time of the query.
+| {check-mark}
 
 | connectComponentsPlanner=idp
 | Use the cost based IDP search algorithm when combining sub-plans.
-
-Note that using this option can significantly increase the planning time of the query.
+[NOTE]
+Using this option can significantly _increase_ the planning time of the query but usually finds better plans.
```

Comment:
```suggestion
[NOTE]
====
Using this option can significantly _increase_ the planning time of the query but usually finds better plans.
====
```

Reply:
When I use this formating, the note doesn't render at all.